app/components/template-viewer/TemplateItem.jsx

```diff
@@ -0,0 +1,12 @@
+var TemplateItem = React.createClass({
+
+    render: function() {
+        return <div className="template-item col-md-10 col-md-offset-1">
```

Comment:
2 line spacing. @wannabeCitizen we setup editor config to automatically deal with these settings. heres the plugin for sublime https://github.com/sindresorhus/editorconfig-sublime